Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -12,29 +12,28 @@ class PayPalViewModel: ObservableObject {
     }
 
     @Published private(set) var state = State.initial
-    private var accessToken = ""
     private var payPalClient: PayPalNativeCheckoutClient?
     private var shippingPreference: OrderApplicationContext.ShippingPreference = .noShipping
     private var orderID = ""
+    private var clientID = ""
 
-    func getAccessToken() {
-        state = .loading(content: "Getting access token")
+    func getClientID() {
+        state = .loading(content: "Getting clientID")
         Task {
-            guard let token = await getAccessToken() else {
-                publishStateToMainThread(.mainContent(title: "Access Token", content: accessToken, flowComplete: false))
+            guard let token = await getClientID() else {
```

Comment:
```suggestion
            guard let clientID = await getClientID() else {
```

Reply:
addressed in 2813be8fab3d5b7d0f2c0c6787ef7ce7d0c141c1